preliz/distributions/discrete.py

```diff
@@ -116,6 +116,92 @@ def _fit_mle(self, sample):
         optimize_ml(self, sample)
 
 
+class BetaBinomial(Discrete):
+    R"""
+    Beta-binomial distribution.
+
+    Equivalent to binomial random variable with success probability
+    drawn from a beta distribution.
+
+    The pmf of this distribution is
+
+    .. math::
+
+       f(x \mid \alpha, \beta, n) =
+           \binom{n}{x}
+           \frac{B(x + \alpha, n - x + \beta)}{B(\alpha, \beta)}
+
+    .. plot::
+        :context: close-figs
+
+        import arviz as az
+        from preliz import BetaBinomial
+        az.style.use('arviz-white')
+        alphas = [0.5, 1, 2.3]
+        betas = [0.5, 1, 2]
+        n = 10
+        for a, b in zip(alphas, betas):
+            BetaBinomial(a, b, n).plot_pdf()
+
+    ========  =================================================================
+    Support   :math:`x \in \{0, 1, \ldots, n\}`
+    Mean      :math:`n \dfrac{\alpha}{\alpha + \beta}`
+    Variance  :math:`\dfrac{n \alpha \beta (\alpha+\beta+n)}{(\alpha+\beta)^2 (\alpha+\beta+1)}`
+    ========  =================================================================
+
+    Parameters
+    ----------
+    n : int
+        Number of Bernoulli trials (n >= 0).
+    alpha : float
+        alpha > 0.
+    beta : float
+        beta > 0.
+    """
+
+    def __init__(self, alpha=None, beta=None, n=10):
```

Comment:
```suggestion
    def __init__(self, alpha=None, beta=None, n=None):
```
`n` should be None, otherwise, for functions like maxent and quartile it will remain fixed